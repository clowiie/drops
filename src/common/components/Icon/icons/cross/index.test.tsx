import React from 'react'

import { render } from '@testing-library/react'

import * as CrossIconType from '.'

describe('CrossIcon', () => {
  const { default: CrossIcon } = require('.') as typeof CrossIconType

  describe('snapshots', () => {
    it('should match snapshots', () => {
      const { container } = render(<CrossIcon size={16} />)

      expect(container.firstChild).toMatchSnapshot()
    })
  })
})
