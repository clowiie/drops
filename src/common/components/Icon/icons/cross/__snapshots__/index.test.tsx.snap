// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`CrossIcon snapshots should match snapshots 1`] = `
<svg
  fill="none"
  height="16"
  viewBox="0 0 16 16"
  xmlns="http://www.w3.org/2000/svg"
>
  <path
    clip-rule="evenodd"
    d="M1.6129 0.209705L1.70711 0.292893L8 6.585L14.2929 0.292893C14.6834 -0.0976311 15.3166 -0.0976311 15.7071 0.292893C16.0676 0.653377 16.0953 1.22061 15.7903 1.6129L15.7071 1.70711L9.415 8L15.7071 14.2929C16.0976 14.6834 16.0976 15.3166 15.7071 15.7071C15.3466 16.0676 14.7794 16.0953 14.3871 15.7903L14.2929 15.7071L8 9.415L1.70711 15.7071C1.31658 16.0976 0.683418 16.0976 0.292893 15.7071C-0.0675907 15.3466 -0.0953203 14.7794 0.209705 14.3871L0.292893 14.2929L6.585 8L0.292893 1.70711C-0.0976311 1.31658 -0.0976311 0.683418 0.292893 0.292893C0.653377 -0.0675907 1.22061 -0.0953203 1.6129 0.209705Z"
    fill="currentColor"
    fill-rule="evenodd"
  />
</svg>
`;
